wrangler-core/src/main/java/io/cdap/wrangler/executor/RecipePipelineExecutor.java

```diff
@@ -161,4 +187,16 @@ private List<Directive> getDirectives() throws RecipeException {
     }
     return directives;
   }
+
+  public static Row getRowUnion(List<Row> rows) {
```

Comment:
I would not do it, but just feed everything to `addDirectiveField` You are essentially do the same here and there